### DISCUSSION TITLE
What is a alternative for Pino in Bun?

### DISCUSSION THREAD

**Discussion by krutoo** [oven-sh/bun#2405] [Q&A]
Hi. In my Node.js application I use [pino](https://github.com/pinojs/pino) for logging events.

I want to try to replace Node.js to Bun. Is there some alternative for `pino` in Bun?

**Comment by hasanyasin** [comment]
I just did this in an empty folder:

```
bun install pino vm
```

Then added this file by simply copying a fragment from pino repo README:

```
// try-pino.js
const logger = require('pino')()

logger.info('hello world')

const child = logger.child({ a: 'property' })
child.info('hello child!')
```

And then this works without any issues:
```
bun try-pino.js
```

Did you face any issues?
